android/app/src/main/jni/main.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "string_script.inc"
+
+#include <android/log.h>
+#include <android_native_app_glue.h>
+#include <cassert>
+#include <vector>
+
+extern "C" {
+#include "config.h"
+#include <vkrunner/vkrunner.h>
```

Comment:
I think angle brackets should only be used for system includes like iostream and stdio.h.

It's not required, but I think it's good style.  You really are looking for these headers from the filesystem.